### PR TITLE
core: libs: commonwealth: settings: pykson: Save in pretty json

### LINKED ISSUE #882: core: commonwealth: Save json pretty
### Bug description

Json file is not pretty

### Steps to reproduce

open json file

### Primary pain point(s)

it's not pretty

### Prerequisites

- [X] I have checked to make sure that a similar request has not already been filed or fixed.

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix #882

### CODE REVIEW

File: core/libs/commonwealth/commonwealth/settings/bases/pykson_base.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def save(self, file_path: pathlib.Path) -> None:
 
         with open(file_path, "w", encoding="utf-8") as settings_file:
             logger.debug(f"Saving settings on: {file_path}")
-            settings_file.write(Pykson().to_json(self))
+            settings_file.write(json.dumps(json.loads(Pykson().to_json(self)), indent=4))
```
Comment (joaoantoniocardoso) [inline core/libs/commonwealth/commonwealth/settings/bases/pykson_base.py:95]: Just checking, have you tried to use the `indent` parameter of [`Pykson::to_json()`](https://github.com/sinarezaei/pykson/blob/master/pykson/__init__.py#L1154)?

Comment (patrickelectric) [reply core/libs/commonwealth/commonwealth/settings/bases/pykson_base.py:95]: Thanks! I'll update it.

File: core/libs/commonwealth/commonwealth/settings/bases/pykson_base.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def save(self, file_path: pathlib.Path) -> None:
 
         with open(file_path, "w", encoding="utf-8") as settings_file:
             logger.debug(f"Saving settings on: {file_path}")
-            settings_file.write(Pykson().to_json(self))
+            settings_file.write(Pykson().to_json(self, base_indent=4))
```
Comment (joaoantoniocardoso) [inline core/libs/commonwealth/commonwealth/settings/bases/pykson_base.py:95]: uhg, maybe our version doesn't have base_indent? Also, what is the difference between `base_indent` and `indent`?

Comment (patrickelectric) [reply core/libs/commonwealth/commonwealth/settings/bases/pykson_base.py]: Thanks! But we have a fork of pykson that is full of changes that are not merged and the project appears to be dead at the moment. We are moving to pydantic, and we should move all projects, removing pykson in the end. This just adds a minor quality for us when checking logs. 

https://github.com/sinarezaei/pykson/pull/19

**Pull request merged**